vnext/Microsoft.ReactNative/Views/cppwinrt/DynamicAutomationPeer.idl

```diff
@@ -61,7 +61,8 @@ namespace Microsoft.ReactNative
     Min = 0,
     Max,
     Now,
-    Text
+    Text,
+    CountValues
```

Comment:
i think this is an antipattern for winrt, in future versions CountValues will have a different value, meaning that code compiled against one version of this and the framework using the new value, will conflict